Gems/EMotionFX/Code/Source/Editor/Plugins/ColliderWidgets/SimulatedObjectColliderWidget.cpp

```diff
@@ -123,6 +124,12 @@ namespace EMotionFX
 
     void SimulatedObjectColliderWidget::InternalReinit()
     {
+        if (QLatin1String("SimulatedObjects") == EMStudio::GetMainWindow()->GetCurrentLayoutName())
```

Comment:
The thought was to make the SimulatedObjects Contents always available when working in the specific 'Mode'.
Maybe it would instead be more wise to move the texts and buttons into the "Simulated Object" window instead.
and only show the colliders in the inspector.

Reply:
Yeah, I'd rather have the content showing functionality tied to a widget instead of tied to a "mode". Especially not tied it with the name of "layout", which is easy to cause problem later.

Reply:
Has been taken care of. See attached videoi